rpcs3/Emu/Cell/SPULLVMRecompiler.cpp

```diff
@@ -3503,7 +3503,7 @@ class spu_llvm_recompiler : public spu_recompiler_base, public cpu_translator
 			return ch->get_count();
 		};
 
-		return m_ir->CreateXor(call("wait_on_spu_channel", +wait_on_channel, m_thread, _ptr<u64>(m_thread, off), m_ir->getInt32(!inv)), m_ir->getInt32(inv));
```

Comment:
What warning does this fix?

Reply:
Something weird with bitwise comparison after changing to u32